ansible_rulebook/vault.py

```
@@ -69,24 +69,49 @@ def __init__(
+                "w+t", suffix=".vaultpw", delete=False
+            )
+            secret_file.write(self.secret)
+            secret_file.flush()
+            secret_file.close()
+            cmd = cmd.replace(
+                "--ask-vault-pass",
+                f"--vault-password-file {secret_file.name}",
+            )
+
+        try:
+            result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)